packages/glib/build.sh

```diff
@@ -162,12 +162,13 @@ termux_step_post_massage() {
 }
 
 termux_step_create_debscripts() {
-	for i in $(test "$TERMUX_PACKAGE_FORMAT" != "pacman" && echo postinst) postrm triggers; do
+	for i in postinst prerm triggers; do
 		sed \
 			"s|@TERMUX_PREFIX@|${TERMUX_PREFIX}|g" \
 			"${TERMUX_PKG_BUILDER_DIR}/hooks/${i}.in" > ./${i}
 		chmod 755 ./${i}
 	done
+	[[ "$TERMUX_PACKAGE_FORMAT" != "pacman" ]] || echo "post_install" > postupg
```

Comment:
```suggestion
	[[ "$TERMUX_PACKAGE_FORMAT" == "pacman" ]] && echo "post_install" > postupg
```
There is no reason to confuse people when only pacman use this

Reply:
I second this.